src/Time.java

```diff
@@ -0,0 +1,10 @@
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Time {
+    public static void main(String[] args) {
+        SimpleDateFormat formater = new SimpleDateFormat ("HH:mm:ss D ww");
+        Date date = new Date(1212121212121L);
```

Comment:
хитро но неправильно. Данный конструктор принимает милисекунды к созданию а в задании есть условие про секунды
Вставка из документации

`/**
     * Allocates a {@code Date} object and initializes it to
     * represent the specified number of milliseconds since the
     * standard base time known as "the epoch", namely January 1,
     * 1970, 00:00:00 GMT.
     *
     * @param   date   the milliseconds since January 1, 1970, 00:00:00 GMT.
     * @see     java.lang.System#currentTimeMillis()
     */
    public Date(long date) {
        fastTime = date;
    }`